lib/config.js

```diff
@@ -1,150 +1,193 @@
 'use strict'
 
+const Immutable = require('immutable')
 const url = require('url')
 const tweetnacl = require('tweetnacl')
 
+function isRunningTests () {
+  return process.env.NODE_ENV.match(/unit|test/) &&
+    (process.argv[0].endsWith('mocha') ||
+    (process.argv.length > 1 && process.argv[0].endsWith('node') &&
+     process.argv[1].endsWith('mocha')))
+}
+
 /**
- * Config management helper class.
+ * Parse a boolean config variable.
  *
- * @example
- * class MyConfig extends Config {
- *   constructor () {
- *     super('example')
- *     this.parseServerConfig()
- *     this.parseDatabaseConfig()
- *   }
- * }
+ * Environment variables are passed in as strings, but this function can turn
+ * values like `undefined`, `''`, `'0'` and `'false'` into `false`.
```

Comment:
Should we take this opportunity to tighten this up? https://github.com/interledger/five-bells-ledger/commit/56e85c3e477c63333965eb17505d8e49d8cc6610#commitcomment-15468958